src/styles/print.css

```diff
@@ -34,17 +34,22 @@
     display: none;
   }
   .h5p-course-presentation .h5p-slides-wrapper .h5p-slide.doprint {
-    display: block;
+    display: inline-flex;
+    flex-flow: row nowrap;
     position: relative;
     -webkit-transform: none;
     transform: none;
 
     /* Adding this border makes it work in FF 39 on Linux. I have no
        Idea why, but please don't remove it !! */
     border: 1px solid #ddd;
-    page-break-inside: avoid;
+    break-inside: avoid;
+    break-after: auto;
```

Comment:
works with inline-flex but not block. Still create some bugs but do not get stuck on the video